app/controllers/seances_controller.rb

```diff
@@ -1,7 +1,10 @@
 class SeancesController < ApplicationController
   # GET /seances
   def index
-    @seances = Seance.includes(:movie, :hall).all
+    @seances = Seance
+               .includes(:movie)
+               .where(starts_at: DateTime.current..Date.today + 7.days)
```

Comment:
This might be simplified to:
```suggestion
               .where(starts_at: DateTime.current..7.days.from_now)
```